plugins/woocommerce/includes/rest-api/Controllers/Version2/class-wc-rest-shipping-zone-methods-v2-controller.php

```diff
@@ -352,7 +352,7 @@ public function prepare_item_for_response( $item, $request ) {
 		$method = array(
 			'id'                 => $item->instance_id,
 			'instance_id'        => $item->instance_id,
-			'title'              => $item->instance_settings['title'],
+			'title'              => isset( $item->instance_settings['title'] ) ? $item->instance_settings['title'] : $item->method_title,
```

Comment:
You can null coalesce this
```suggestion
			'title'              => $item->instance_settings['title'] ?? $item->method_title,
```

Reply:
https://github.com/woocommerce/woocommerce/pull/57179/commits/1b9239762fee4d24dde5cfb107cfe9c86fd0b9d9